build.sbt

```diff
@@ -92,12 +93,9 @@ lazy val commonScalacOptions = Seq(
   "-unchecked",
   "-Xfatal-warnings",
   "-Xlint",
-  "-Yno-adapted-args",
   "-Ywarn-dead-code",
   "-Ywarn-numeric-widen",
-  "-Ywarn-value-discard",
-  "-Xfuture",
-  "-Xexperimental"
```

Comment:
`-Xexperimental` is required in order to compile project in Scala 2.11. This flag enables support for Single Abstract Methods.

We can either conditionally enable this flag only for Scala 2.11 (preferred) or turn all SAM expressions into legacy syntax with `new XXX[...] { ... }` (less preferred).

Reply:
I added `-Xexperimental` for 2.11. Now it compiles for 2.11.